pkg/services/anonymous/anonimpl/impl.go

```diff
@@ -57,17 +59,23 @@ func ProvideAnonymousDeviceService(remoteCache remotecache.CacheStorage, usageSt
 }
 
 func (a *AnonDeviceService) usageStatFn(ctx context.Context) (map[string]interface{}, error) {
-	sessionCount, err := a.remoteCache.Count(ctx, anonCachePrefix)
+	anonDeviceCount, err := a.remoteCache.Count(ctx, string(anonymous.AnonDevice))
+	if err != nil {
+		return nil, nil
+	}
+
+	authedDeviceCount, err := a.remoteCache.Count(ctx, string(anonymous.AuthedDevice))
 	if err != nil {
 		return nil, nil
 	}
 
 	return map[string]interface{}{
-		"stats.anonymous.session.count": sessionCount,
+		"stats.anonymous.session.count": anonDeviceCount, // keep session for legacy data
```

Comment:
Thought: could we double-write and have a `stats.anonymous.device.count` metric as well? We could then get rid of the `stats.anonymous.session.count` after 30 days, and have more consistent metrics and references.

Reply:
It's better to keep the old name in a case like this where we are not changing the semantics of the metric, since otherwise we still have old versions reporting the old name while new versions report the new name. On the flip side, when we do change the meaning of the metric we need to make sure we change the name!

Reply:
user session is an existent term already, for example `stats.active_sessions.count` already exists, I think it would be more confusing than consistent